packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -211,6 +215,17 @@ class ArtifactStore {
     return _engineRevision;
   }
 
+  static String _expectedEngineRevision;
+
+  static String get expectedEngineRevision {
+    if (_expectedEngineRevision == null) {
+      File revisionFile = new File(path.join(flutterRoot, 'packages', 'flutter', 'packages', 'sky_engine', 'REVISION'));
```

Comment:
this should read the path out of the .packages file instead (or add a TODO to do it)